interfaces/builtin/steam_support.go

```diff
@@ -395,6 +247,34 @@ type steamSupportInterface struct {
 	commonInterface
 }
 
+func (iface *steamSupportInterface) AppArmorConnectedPlug(spec *apparmor.Specification, plug *interfaces.ConnectedPlug, slot *interfaces.ConnectedSlot) error {
+	// if apparmor supports "allow all" then use it. This allows not updating
+	// the supported features list as new features are added.
+	if apparmor_sandbox.ProbedLevel() != apparmor_sandbox.Unsupported {
+		features, err := apparmor_sandbox.ParserFeatures()
+		if err != nil {
+			return err
+		}
+		if strutil.ListContains(features, "allow-all") {
+			spec.AddSnippet(steamSupportConnectedPlugAppArmorAll)
+		} else {
+			spec.AddSnippet(steamSupportConnectedPlugAppArmor)
+			if strutil.ListContains(features, "mqueue") {
+				spec.AddSnippet("allow mqueue,\n")
```

Comment:
By convention we seem to define these snippets as global vars rather than inline - see https://github.com/snapcore/snapd/blob/master/interfaces/builtin/docker_support.go#L65 for an example

Reply:
Yeah but at some point the cost of indirection (mental cost) may outweigh the benefit.